diff --git a/src/ipa/libipa/camera_sensor_helper.cpp b/src/ipa/libipa/camera_sensor_helper.cpp
index dcd69d9..932c4f3 100644
--- a/src/ipa/libipa/camera_sensor_helper.cpp
+++ b/src/ipa/libipa/camera_sensor_helper.cpp
@@ -497,6 +497,18 @@ public:
 };
 REGISTER_CAMERA_SENSOR_HELPER("gc08a3", CameraSensorHelperGc08a3)
 
+class CameraSensorHelperGc5035 : public CameraSensorHelper
+{
+public:
+	CameraSensorHelperGc5035()
+	{
+		/* From driver behavior: 64 at 10bits. */
+		blackLevel_ = 4096;
+		gain_ = AnalogueGainLinear{ 1, 0, 0, 256 };
+	}
+};
+REGISTER_CAMERA_SENSOR_HELPER("gc5035", CameraSensorHelperGc5035)
+
 class CameraSensorHelperImx214 : public CameraSensorHelper
 {
 public:
@@ -689,6 +701,18 @@ public:
 };
 REGISTER_CAMERA_SENSOR_HELPER("ov5675", CameraSensorHelperOv5675)
 
+class CameraSensorHelperOv5678 : public CameraSensorHelper
+{
+public:
+	CameraSensorHelperOv5678()
+	{
+		/* Same gain coding and pedestal convention as OV5675 class. */
+		blackLevel_ = 4096;
+		gain_ = AnalogueGainLinear{ 1, 0, 0, 128 };
+	}
+};
+REGISTER_CAMERA_SENSOR_HELPER("ov5678", CameraSensorHelperOv5678)
+
 class CameraSensorHelperOv5693 : public CameraSensorHelper
 {
 public:
