--- ipu-bridge.original.c	2026-04-18 19:52:31.741813983 -0700
+++ ipu-bridge.c	2026-04-18 19:53:22.321490324 -0700
@@ -48,6 +48,8 @@
  * Please keep the list sorted by ACPI HID.
  */
 static const struct ipu_sensor_config ipu_supported_sensors[] = {
+	/* GalaxyCore GC5035 */
+	IPU_SENSOR_CONFIG("GCTI5035", 1, 438000000),
  	/* Himax HM11B1 */
  	IPU_SENSOR_CONFIG("HIMX11B1", 1, 384000000),
  	/* Himax HM2170 */
@@ -88,6 +90,8 @@
 	IPU_SENSOR_CONFIG("OVTIDB10", 1, 560000000),
 	/* Omnivision OV2680 */
 	IPU_SENSOR_CONFIG("OVTI2680", 1, 331200000),
+	/* Omnivision OV5678 */
+	IPU_SENSOR_CONFIG("OVTI5678", 1, 422400000),
 	/* Omnivision OV8856 */
 	IPU_SENSOR_CONFIG("OVTI8856", 3, 180000000, 360000000, 720000000),
 	/* Toshiba T4KA3 */
